packages/common/react-components/src/components/TreeView/TreeView.tsx

```diff
@@ -5,7 +5,8 @@
 import { CaretDown, CaretRight } from 'phosphor-react';
 import React, { FC, ReactNode, useState } from 'react';
 
-import { getSize, mx } from '@dxos/react-components';
+import { getSize } from '../../styles';
+import { mx } from '../../util';
 
```

Comment:
I believe @richburdon was going to make this move, make sure these changes will gel with the ones he intends to make.

Reply:
Radix provides a `Collapsible` component which would do a decent job of patching-in good interactivity here. This component should eventually be refactored to use it: https://www.radix-ui.com/docs/primitives/components/collapsible